test/src/main/java/org/apache/accumulo/test/ScanConsistencyIT.java

```diff
@@ -104,69 +150,97 @@ public void testConcurrentScanConsistency() throws Exception {
+    int numWriteTask = random.nextInt(10) + 1;
+    int numsScanTask = random.nextInt(10) + 1;
 
-      for (int i = 0; i < numWriteTask; i++) {
-        writeTasks.add(executor.submit(new WriteTask(testContext)));
-      }
+    for (int i = 0; i < numWriteTask; i++) {
+      writeTasks.add(executor.submit(new WriteTask(testContext)));
+    }
 
-      for (int i = 0; i < numsScanTask; i++) {
-        scanTasks.add(executor.submit(new ScanTask(testContext)));
-      }
+    for (int i = 0; i < numsScanTask; i++) {
+      scanTasks.add(executor.submit(new ScanTask(testContext)));
+    }
 
-      var tableOpsTask = executor.submit(new TableOpsTask(testContext));
+    var tableOpsTask = executor.submit(new TableOpsTask(testContext));
 
-      // let the concurrent mayhem run for a bit
-      Thread.sleep(60000);
+    // let the concurrent mayhem run for a bit
+    Thread.sleep(60000);
 
-      // let the threads know to exit
-      testContext.keepRunning.set(false);
+    // let the threads know to exit
+    testContext.keepRunning.set(false);
 
-      for (Future<WriteStats> writeTask : writeTasks) {
-        var stats = writeTask.get();
-        log.debug(String.format("Wrote:%,d Bulk imported:%,d Deleted:%,d Bulk deleted:%,d",
-            stats.written, stats.bulkImported, stats.deleted, stats.bulkDeleted));
-        assertTrue(stats.written + stats.bulkImported > 0);
-        assertTrue(stats.deleted + stats.bulkDeleted > 0);
-      }
+    for (Future<WriteStats> writeTask : writeTasks) {
+      var stats = writeTask.get();
+      log.debug(String.format("Wrote:%,d Bulk imported:%,d Deleted:%,d Bulk deleted:%,d",
+          stats.written, stats.bulkImported, stats.deleted, stats.bulkDeleted));
+      checkTrue(stats.written + stats.bulkImported > 0);
+      checkTrue(stats.deleted + stats.bulkDeleted > 0);
+    }
 
-      for (Future<ScanStats> scanTask : scanTasks) {
-        var stats = scanTask.get();
-        log.debug(String.format("Scanned:%,d verified:%,d", stats.scanned, stats.verified));
-        assertTrue(stats.verified > 0);
-        // These scans were running concurrently with writes, so a scan will see more data than what
-        // was written before the scan started.
-        assertTrue(stats.scanned > stats.verified);
-      }
+    for (Future<ScanStats> scanTask : scanTasks) {
+      var stats = scanTask.get();
+      log.debug(String.format("Scanned:%,d verified:%,d", stats.scanned, stats.verified));
+      checkTrue(stats.verified > 0);
+      // These scans were running concurrently with writes, so a scan will see more data than what
+      // was written before the scan started.
+      checkTrue(stats.scanned > stats.verified);
+    }
+
+    log.debug(tableOpsTask.get());
+
+    var stats1 = scanData(testContext, random, new Range(), false);
+    var stats2 = scanData(testContext, random, new Range(), true);
+    var stats3 = batchScanData(testContext, new Range());
+    log.debug(
```

Comment:
When running this test standalone, it would be nice to see these logs at the `info` level instead of `debug`. Its not much of a hassle to move my instance to display the debug logs, bug when I do they get drowned out by a bunch of other debug logs.

I know these logs were not added in this PR but they could be changed since its related to the new context that this code is run.

Reply:
> bug when I do they get drowned out by a bunch of other debug logs.

Should be able to enable debug logging just for this class in log4j2 config.

Changing the log level to info in the code should be ok too.